submissions/Serge044/Classic-Frogger-Game/app.js

```diff
@@ -0,0 +1,151 @@
+const Enemy = function (x, y) {
+  this.width = 75;
+  this.x = x;
+  this.y = y;
+  this.border = { left: -100, right: 505 };
+  this.speed = getRandomNum(200, 300);
+  this.sprite = "images/tank-icon.png";
+};
+
+Enemy.prototype.update = function (dt) {
+  if (this.x < this.border.right) {
+    this.x += dt * this.speed + Math.random();
+  } else {
+    this.speed = getRandomNum(200, 300);
+    this.x = this.border.left;
+  }
+};
+
+function getRandomNum(min, max) {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+Enemy.prototype.render = function () {
+  ctx.drawImage(Resources.get(this.sprite), this.x, this.y);
+};
+
+const Player = function () {
+  this.width = 75;
+  this.x = 202;
+  this.y = 400;
+  this.stepX = 101;
```

Comment:
Any correlation between `stepX` and `x`? I feel, if dimensions of game characters and board tiles change we'd need to fix numbers in many places. Use computer to make calculations for you.